extensions/ql-vscode/src/view/variant-analysis/__tests__/VariantAnalysisStats.spec.tsx

```diff
@@ -100,4 +101,22 @@ describe(VariantAnalysisStats.name, () => {
     expect(screen.getByText('Succeeded')).toBeInTheDocument();
     expect(screen.queryByText('Succeeded warnings')).not.toBeInTheDocument();
   });
+
+  it('renders the duration when it is less than a second', () => {
```

Comment:
Is it worth adding one more test case for when the duration shouldn't be rendered (i.e. when `completedAt` isn't available).?

Reply:
I'll add a test case for it, but it's somewhat hard to test this since there's not a clearly correct text that should be shown. `-` will be shown when there is no duration, but this will also be shown when there are no results. I've now added some simple test cases that test that the correct number of `-` are shown, but this might be somewhat less robust again changes in this component.